tests/DocumentationExamplesTest.php

```diff
@@ -1540,6 +1540,85 @@ public function testCausalConsistency()
         ob_end_clean();
     }
 
+    /**
+     * @doesNotPerformAssertions
+     */
+    public function testWithTransactionExample()
+    {
+        $this->skipIfTransactionsAreNotSupported();
+
+        $uriString = static::getUri(true);
+
+        // phpcs:disable SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly
+        // Start Transactions withTxn API Example 1
+        /*
+         * For a replica set, include the replica set name and a seedlist of the members in the URI string; e.g.
+         * uriString = 'mongodb://mongodb0.example.com:27017,mongodb1.example.com:27017/admin?replicaSet=myRepl'
+         * For a sharded cluster, connect to the mongos instances; e.g.
+         * uriString = 'mongodb://mongos0.example.com:27017,mongos1.example.com:27017/admin'
+         */
+
+        $client = new \MongoDB\Client($uriString);
+
+        // Prerequisite: Create collections. CRUD operations in transactions must be on existing collections.
+        $client->selectCollection(
+            'test',
+            'foo',
+            [
+                'readConcern' => new \MongoDB\Driver\ReadConcern(\MongoDB\Driver\ReadConcern::MAJORITY),
+                'writeConcern' => new \MongoDB\Driver\WriteConcern(\MongoDB\Driver\WriteConcern::MAJORITY, 1000),
+            ]
+        )->insertOne(['abc' => 0]);
+
+        $client->selectCollection(
+            'test',
+            'bar',
+            [
+                'readConcern' => new \MongoDB\Driver\ReadConcern(\MongoDB\Driver\ReadConcern::MAJORITY),
+                'writeConcern' => new \MongoDB\Driver\WriteConcern(\MongoDB\Driver\WriteConcern::MAJORITY, 1000),
+            ]
+        )->insertOne(['xyz' => 0]);
+
+        // Step 1: Define the callback that specifies the sequence of operations to perform inside the transactions.
+
+        $callback = function (\MongoDB\Driver\Session $session) use ($client) {
+            $client->selectCollection(
```

Comment:
The clients created in the callback also use `mydb1.foo` and `mydb2.bar` as their namespaces, respectively. They also don't take any read or write concern options.